app/models/miq_database.rb

```diff
@@ -72,14 +61,8 @@ def self.adapter
   end
 
   def verify_credentials(auth_type = nil, _options = {})
```

Comment:
We can probably get rid of this and the `include AuthenticationMixin` unless you want to do that in a followup PR